src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -517,13 +521,52 @@ const OfflineTokenRow = ({ offlineToken, onValueChanged, formValidationFailed })
                 </FlexItem>
             </Flex>,
         },
+        EXPIRED: {
+            option: "default",
+            message: <Flex>
+                <FlexItem grow={{ default: 'grow' }}>{_("Old token expired") + " "}</FlexItem>
+                <FlexItem>
+                    { link }
+                    {" " + _("Then copy and paste it above.")}
+                </FlexItem>
+            </Flex>,
+        },
         SUCCESS: {
             option: "success",
             message: _("Valid token"),
         },
     };
 
     const [validationState, setValidationState] = useState(validationStates.DEFAULT);
+    const [disabled, setDisabled] = useState(false);
+
+    useEffect(() => {
+        loadOfflineToken((token) => {
```

Comment:
This functionality needs a test. Please log out, back in, and start installing a RHEL image. The dialog should get a valid token immediately. You should also be able to test expiry -- either by dropping it from the fake rhsm (cleaner), or by changing the value in the browser with a direct `localStorage.setItem()` from the tests (quick&dirty, but may be easier).